external/CMakeLists.txt

```diff
@@ -359,6 +359,57 @@ if ((NOT LIBDSDCC_FOUND OR LIBDSDCC_EXTERNAL) AND LIBMBE_FOUND)
     endif ()
 endif ((NOT LIBDSDCC_FOUND OR LIBDSDCC_EXTERNAL) AND LIBMBE_FOUND)
 
+# For APT demodulator
+set(APT_LIBRARIES "${SDRANGEL_BINARY_LIB_DIR}/apt.lib" CACHE INTERNAL "")
```

Comment:
This is for Windows but the equivalent is missing for Linux. I think this is why the build for Ubuntu 20.04 fails.